example-app/src/main/java/com/adyen/checkout/example/data/storage/CardAddressMode.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 8/8/2023.
+ */
+
+package com.adyen.checkout.example.data.storage
+
+enum class CardAddressMode {
```

Comment:
The enums we save in shared prefs should have `@Keep` on them, so it keeps working with obfuscation

Reply:
Good one 👍